e2e-tests/custom-login/conf.js

```diff
@@ -0,0 +1,68 @@
+/*!
+ * Copyright (c) 2015-2018, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+/* eslint import/no-unresolved:0, import/no-extraneous-dependencies:0, no-console:0 */
+/* global jasmine */
+const jasmineReporters = require('jasmine-reporters');
+const daemonUtil = require('../tools/daemon-util');
+
+const promises = Promise.all([
+  daemonUtil.startCustomLoginServer(),
+  daemonUtil.startResourceServer()
+]);
+
+const config = {
+  // Set the following env vars to match your test environment
+  // Note the USERNAME should be of the form "username@email.com"
+  params: {
+    login: {
+      username: process.env.USERNAME,
+      password: process.env.PASSWORD,
+      email: process.env.USERNAME,
+    },
+    appRoot: 'http://localhost:8080'
```

Comment:
Did we decide on making this static instead of configurable? I thought this was also am environment variable.

Reply:
Won't the app always be run on `localhost`?

Reply:
It will, but we should make the `port` variable:

```
appRoot: `http://localhost:${process.env.PORT || 8080}`;
```